modules/deployment-plug-in-cluster.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="deploy-on-cluster_{context}"]
+= Deploy your plug-in on a cluster
+
+After pushing an image with your changes to a registry, you can deploy the plug-in to a cluster.
+
+.Procedure
+
+. Deploy your plug-in to a cluster by instantiating the link:https://github.com/spadgett/console-plugin-template/blob/main/template.yaml[template]. It will run a light-weight nginx HTTP server to serve your plug-in's assets.
```

Comment:
```suggestion
. Deploy your plug-in to a cluster by instantiating the link:https://github.com/spadgett/console-plugin-template/blob/main/template.yaml[template]. It will run a light-weight NGINX HTTP server to serve the assets for your plug-in.
```